sentry-android/src/main/java/com/joshdholtz/sentry/Sentry.java

```diff
@@ -102,6 +103,12 @@ private Sentry() {
 
 	}
 
+	private static void log(String text) {
+		if (debug) {
+			log(text);
```

Comment:
If `debug = true`, `log` will just call itself recursively. Surely you mean `if(debug) { Log.d(text); }`